RFS/src/main/java/org/opensearch/migrations/bulkload/transformers/Transformer_ES_6_8_to_OS_2_11.java

```diff
@@ -38,8 +38,16 @@ public GlobalMetadata transformGlobalMetadata(GlobalMetadata globalData) {
             templatesRoot.fields().forEachRemaining(template -> {
                 var templateCopy = (ObjectNode) template.getValue().deepCopy();
                 var indexTemplate = (Index) () -> templateCopy;
-                transformIndex(indexTemplate, IndexType.TEMPLATE);
-                templates.set(template.getKey(), indexTemplate.getRawJson());
+                try {
+                    transformIndex(indexTemplate, IndexType.TEMPLATE);
+                    templates.set(template.getKey(), indexTemplate.getRawJson());
+                }  catch (Exception e) {
+                    log.atError()
```

Comment:
Do we currently populate anything in the summary to say a transformation didn't work?

Reply:
Good callout - Nope, any items that are 'rejected' in this way will never show up in the UX (So at least there wouldn't be a false positive)

We will need to reorganize the data processing model to handle this cases :(